docs/creating-test-environment.md

```diff
@@ -1,316 +1,199 @@
+
 2. Run the Installation Command
 
-```sh
-make install
-```
+   ```sh
+   make install
+   ```
 
 Ensure the CRDs are installed with `kubectl get crds | grep kyma-project.io`:
 
 ```
-manifests.operator.kyma-project.io        2022-08-18T16:27:21Z
+manifests.component.kyma-project.io        2022-08-18T16:27:21Z
 kymas.operator.kyma-project.io             2022-08-18T16:29:28Z
 moduletemplates.operator.kyma-project.io   2022-08-18T16:29:28Z
 ```
 
-#### Run the Operators
+### 3.3.3 Run the operators
 
-_Note for Remote Clusters: Make sure you run the commands with `KUBECONFIG` set to the KCP Cluster_
+#### 3.3.3.1 Deploy and run operators in cluster
 
-In https://github.com/kyma-project/lifecycle-manager run
+_Note: The order of installation is important due to cross-dependencies in CRDs_
+
+In https://github.com/kyma-project/module-manager/operator run
 
 ```sh
-make run
+# using local registry
+make docker-build docker-push deploy IMG=$IMG_REGISTRY/module-manager:dev
+
+# using remote registry (replace `PR-73` with your desired tag)
+make deploy IMG=eu.gcr.io/kyma-project/module-manager:PR-73
 ```
 
-In https://github.com/kyma-project/module-manager run
+In https://github.com/kyma-project/lifecycle-manager/operator run
 
 ```sh
-make run
-```
+# using local registry
+make docker-build docker-push deploy IMG=$IMG_REGISTRY/lifecycle-manager:dev
 
-### Run in Control Plane
+# using remote registry (replace `PR-122` with your desired tag)
+make deploy IMG=eu.gcr.io/kyma-project/lifecycle-manager:PR-122
+```
 
-_Note: The order of installation is important due to cross-dependencies in CRDs_
+_Note: It could be that you get messages like `no matches for kind "VirtualService" in version "networking.istio.io/v1beta1"`. This is normal if you install the operators in a cluster without a certain dependency. If you do not need this for your test, you can safely ignore it._
 
-In https://github.com/kyma-project/module-manager run
+#### 3.3.3.2 Run on your local host
 
-```sh
-make deploy IMG=eu.gcr.io/kyma-project/module-manager:PR-73
-```
+1. In https://github.com/kyma-project/lifecycle-manager run
 
-_Note: Replace `PR-73` with your desired tag_
+   ```sh
+   make run
+   ```
 
-In https://github.com/kyma-project/lifecycle-manager run
+2. In https://github.com/kyma-project/module-manager run
 
-```sh
-make deploy IMG=eu.gcr.io/kyma-project/lifecycle-manager:PR-122
-```
+   ```sh
+    make run
+   ```
 
-_Note: Replace `PR-122` with your desired tag_
+## 3.4 Start the Kyma installation
 
-_Note: It could be that you get messages like `no matches for kind "VirtualService" in version "networking.istio.io/v1beta1"`. This is normal if you install the operators in a cluster without a certain dependency. If you do not need this for your test, you can safely ignore it._
+_Note for two cluster mode: Make sure you run the commands with `KUBECONFIG` set to the KCP Cluster_
 
```

Comment:
```suggestion

_Note for using Google Artifact Registry: if you use gcloud cli, make sure your local docker config (`~/.docker/config.json`) does not contains `gcloud` `credHelpers` entry, (e.g: `"europe-west3-docker.pkg.dev": "gcloud",`), otherwise this might cause authentication issue for module-manager while fetching remote oci image layer.
```